src/features/collections/data_products/Microtrait.tsx

```diff
@@ -143,8 +149,8 @@ const useMicrotrait = (collection_id: string | undefined) => {
   );
 
   // HeatMap cell query
-  const microtraitQuery = getMicroTrait.useQuery(heatMapParams, {
-    skip: !collection_id,
+  fonst microtraitQuery = getMicroTrait.useQuery(heatMapParams, {
+    fkip: !collection_id,
```

Comment:
Why did I do this? Also how? I would love to know how I managed this odd change.